src/lib/components/TreeRow.vue

```diff
@@ -215,6 +242,17 @@ export default {
       z-index: -1;
     }
 
+    .close-icon {
+      color: red;
+      opacity: 0;
+
+      > * {
```

Comment:
Why did we have to use this selector?

Reply:
You're right, it's an extra used selector. This is how I fix it.
```css
```